test/models/version_test.rb

```diff
@@ -10,4 +10,9 @@ class VersionTest < ActiveSupport::TestCase
     change = versions(:page2_v2).change_from_previous
     assert_not_nil change
   end
+
+  test 'next should get the next version' do
+    nxt = versions(:page1_v1).next
```

Comment:
This is a really minor nit, but we try and avoid most abbreviations. I know `next` is a keyword in Ruby, but how about `url` or `next_url` or something similar?

Reply:
I changed it to `next_version` since the object is a `Version` object. That way it's not confused for an actual url path.

Reply:
Oh, ha, you’re right! I was stuck thinking about the data in the response :P